### PR TITLE
ENH: Add recipes for databroker and datamuxer

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: recipes/databroker/v0.3.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package:
+  name: databroker
+  version: v0.3.0
+
+source:
+  git_url: https://github.com/NSLS-II/databroker
+  git_rev: master
```
Comment (tacaswell) [inline recipes/databroker/v0.3.0/meta.yaml]: This one probably needs the build string override?

File: recipes/datamuxer/dev/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  name: datamuxer
+  version: {{ environ['GIT_DESCRIBE_TAG'] }}.post{{ environ['GIT_DESCRIBE_NUMBER'] }}
+
+source:
+  git_url: https://github.com/NSLS-II/datamuxer
+  git_rev: master
+
+build:
+  number: 0
+  string: {{ environ['GIT_DESCRIBE_TAG'] }}.post{{ environ['GIT_DESCRIBE_NUMBER'] }}
```
Comment (tacaswell) [inline recipes/datamuxer/dev/meta.yaml]: Does this also include the python version?

Comment (ericdill) [inline recipes/datamuxer/dev/meta.yaml]: that is a bad copy/paste

**Comment by ericdill** [comment]
@tacaswell good catch. Fixed that bad build string

File: recipes/databroker/v0.3.0/meta.yaml (hunk 1 above)
Comment (tacaswell) [inline recipes/databroker/v0.3.0/meta.yaml]: This should be pinned to a tag, not to master.  I now understand why I was confused.

Comment (ericdill) [inline recipes/databroker/v0.3.0/meta.yaml]: Yes it should, Thanks for catching this too

**Comment by ericdill** [comment]
@tacaswell another good catch. Pinned the v0.3.0 recipe to actually build the v0.3.0 tag

**Pull request merged**